### ISSUE TITLE
I want to write the user manual in Spanish

### ISSUE THREAD

**Issue by RafaMartorell** [olive-editor/olive#1024]
I would like to be recommended to me which format is the most suitable for it.

I had planned to do it with LibreOffice .odt and then convert it into a downloadable pdf for those interested.

But I could also make a html file without style that could be attached to the Olive website.

Another option is to put it on my free wordpress blog.

Well, if someone provides a recommendation, it will be welcome, the important thing is to write the manual and little by little I will do it, once finished I will do some video tutorials to teach its management and professional techniques of video editing. Some time ago I did it with kdenlive but this editor disappointed me and on top of that an update that promised that it was going to be the seventh wonder has turned out to be a big step backwards ... but well it does not matter, now I want to focus on Olive.

**Comment by Simran-B** [comment]
Great to have you on board!

Documentation is always a moving target, but with the rewrite in progress I think this is not a good time to write a manual. A lot of the content will be obsoleted with the next release, and chances are that there will be more drastic changes in subsequent versions.

My personal format preference would be Markdown, as it can be converted to HTML if necessary or used directly in static-site generators like Jekyll or Hugo. The GitHub Wiki also uses Markdown.

**Comment by RafaMartorell** [comment]
Thank you very much for your response, I take note of your recommendations and I will try to write about basic things that I know will not change in future versions and I will write later on what may change. As I already mentioned, I am an audiovisual technician and I do not understand much about programming and I honestly do not know the recommendations that you indicate but I will look for information about Markdown and this will be the format I will use. At the moment it had begun to realize it in format .odt of LibreOffice with the intention to turn it into pdf.
Thank you very much again and happy to be on board this project that I am really enjoying.

**Comment by Simran-B** [comment]
A variant of Markdown is used right here on GitHub. Markdown is pretty limited but often sufficient and also human-readable when not rendered:

normal text → normal text

&#x2a;italic&#x2a; → *italic*
&#x5f;italic&#x5f; → _italic_

&#x2a;&#x2a;bold&#x2a;&#x2a; → **bold**
&#x5f;&#x5f;bold&#x5f;&#x5f; → __bold__

&#x5b;link label](https&#x3a;//example.org) → [link label](https://example.org)

!&#x5b;image label](http&#x3a;//lorempixel.com/50/30/animals/8/) → ![link label](http://lorempixel.com/50/30/animals/8/)

&nbsp;

&#x60;inline code&#x60; → `inline code`

&nbsp;

&nbsp;&nbsp;&nbsp;&nbsp;code block started by 4 leading spaces

    code block started by 4 leading spaces

&nbsp;

&#x60;&#x60;&#x60;
fenced code block
&#x60;&#x60;&#x60;

```
fenced code block
```

&nbsp;

&gt;blockquote
> blockquote

&nbsp;

&#x2d; list item 1
&#x2d; list item 2

- list item 1
- list item 2

&nbsp;

&#x2a; list item 1
&#x2a; list item 2

* list item 1
* list item 2

&nbsp;

1&#x2e; numbered list item 1
2&#x2e; numbered list item 2

1. numbered list item 1
2. numbered list item 2

&nbsp;

&#x23; headline level 1
# headline level 1

&nbsp;

&#x23;&#x23; headline level 2
## headline level 2

&nbsp;

&#x23;&#x23;&#x23; headline level 3
### headline level 3

&nbsp;

... up to ###### headline level 6. There are also alternatives for the first two levels:

headline level 1
&#x3d;&#x3d;&#x3d;

headline level 1
===

&nbsp;

headline level 2
&#x2d;&#x2d;&#x2d;

headline level 2
---

&nbsp;

Horizontal ruler:

&#x2d;&#x2d;&#x2d;

---

&nbsp;

So far for standard Markdown 😁 
Also see https://daringfireball.net/projects/markdown/basics

Interactive sandbox: https://md2pdf.netlify.com/

**Comment by RafaMartorell** [comment]
Thank you very much for your advice and this brief introduction to the Markdown syntax.

I'm finding it easy to understand and edit with markdown so thank you again.

Although I'm already writing some sections on the page, I thought it would be a space that only I could see, but I see that it is public, so if I did something wrong do not hesitate to delete the content, I made a backup :-)

As you told me, it might be a bit early to write a manual, but at the moment that the next version comes out I will put myself into it.

Currently I will write generic things that I guess will serve me for the new version, and if not, nothing happens, I like this project and I will help as much as I can, but once again I want to say that I am an audiovisual technician and my notions of programming are very low.

Greetings and many thanks.
